neo/io/asciisignalio.py

```diff
@@ -292,7 +291,7 @@ def read_segment(self, lazy=False):
                                            name='Column %d' % i)
                     seg.analogsignals.append(ana_sig)
 
-        seg.create_many_to_one_relationship()
+        seg.check_relationships()
```

Comment:
Is this still needed ? All relationship are now check on the fly no ?

Reply:
It is not needed, but it acts as an extra check, to give confidence that everything is properly linked up.

Once we're sure everything is working, these `check_relationship()` calls could be removed (i.e. in a future PR)

Reply:
OK.